CHANGES/144.feature

```diff
@@ -0,0 +1 @@
+Added worker command.
```

Comment:
Meh, self referencing does not seem to work. But i like the idea of having a changelog without the need to file an issue just to say: This is what i do in the PR anyway.

OTOH, i thought gh handles PRs as a special sort of issues.

It should work however if you do not put the `fixes` in the commit message. Which means we should add more checking for spurious changelog snippets. What if "(no [noissue]) and (no fixes #??)" implies we need one for the PR number?